### PR TITLE
NAS-119165 / 23.10 / Default to Recreate on Launch Docker instead of RollingUpdate

### PR DESCRIPTION
**Pull request by stavros-k**

### CODE REVIEW

**Comment by stavros-k** [comment]
@Qubad786  The issue the user had, can be cause by both the `restartPolicy` and `updateStrategy`.
Let's not default to rolling update, as there are lot of applications that won't work well on updates with this, trying both to read/write the same data.
(eg apps that use sqlite db, or even normal databases like postgres)

File: charts/ix-chart/2212.0.0/questions.yaml [hunk 1]
```diff
@@ -166,11 +166,12 @@ questions:
     label: "Restart Policy"
     group: "Restart Policy"
     schema:
-      hidden: true
       type: string
       default: "OnFailure"
       show_if: [["workloadType", "!=", "Deployment"]]
       enum:
+        - value: "Always"
```
[the remark this answers is not in the document]
Comment (stavros-k) [reply charts/ix-chart/2212.0.0/questions.yaml:190]: Indeed, updated.

[the remark this answers is not in the document]
Comment (stavros-k) [reply charts/ix-chart/2212.0.0/questions.yaml:190]: Re-hidden the jorRestartPolicy as we don't have the workload type visible anyway

[the remark this answers is not in the document]
Comment (stavros-k) [reply charts/ix-chart/2212.0.0/questions.yaml]: Deployment workloads only support `Always` as restartPolicy.

**Review by Qubad786 — APPROVED** [review]

**Pull request merged**